src/EnergyPlus/CurveManager.cc

```diff
@@ -610,6 +609,8 @@ namespace Curve {
         int IOStatus;                    // Used in GetObjectItem
         std::string CurrentModuleObject; // for ease in renaming.
 
+        if (!state.dataCurveManager->GetCurvesInputFlag) return;
+        state.dataCurveManager->GetCurvesInputFlag = false;
```

Comment:
![image](https://github.com/user-attachments/assets/f7c7b168-2367-43f2-bede-374c03fbd6b7)

Reply:
:laughing: :rofl: :smile:   This is excellent @rraustad !

Reply:
Wow.